style.css

```diff
@@ -0,0 +1,76 @@
+.title {
+  font-size: 10px;
+}
+
+body {
+  font-family: arial, sans-serif;;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  background: #008aff;
+}
+
+.nav a {
+  color: #fff;
+  padding: 15px 10px;
+  text-decoration: none;
+  text-align: center;
+}
+
+.nav a:hover {
+  background: #5c8db7;
+}
+
+@media all and (max-width: 800px) {
+.nav {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ad/pullRequest?prid=5498083)